controller/api/destination/server_test.go

```diff
@@ -917,8 +944,8 @@ func TestGetProfiles(t *testing.T) {
 		if first.Endpoint.ProtocolHint == nil {
 			t.Fatalf("Expected protocol hint but found none")
 		}
-		if first.Endpoint.ProtocolHint.GetOpaqueTransport().InboundPort != 4143 {
-			t.Fatalf("Expected pod to support opaque traffic on port 4143")
+		if first.Endpoint.ProtocolHint.GetOpaqueTransport().InboundPort != opaquePort {
+			t.Fatalf("Expected pod to support opaque traffic on port %d", opaquePort)
```

Comment:
`InboundPort` should be the application port, not the proxy port. I'm not sure how we weren't catching this before.